action_server/docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
   from the action server notifying users about it if the old directory exists).
 - The `SEMA4AI_HOME` environment variable can now be used to set a different home
   folder to store the `action-server` directory information.
+- `new` command allows to choose a template from a predefined list now. Command will list available templates,
+  and prompt user to choose one. 
+  - `--template` argument has been, allowing to specify a concrete template when running the command (when specified,
```

Comment:
It feels like something is missing in this phrase...
`--template` argument has been, allowing`